pandas/_libs/reduction.pyx

```diff
@@ -448,28 +448,22 @@ cdef class Slider:
             values = values.copy()
 
         self.values = values
+        self.orig_buf = buf
```

Comment:
Do the tests work if you copy here?

Reply:
nope

Reply:
Strange. At initial glance I would think what you've done is equivalent...

Reply:
Glad I'm not the only one.  Do you have another strategy in mind for addressing the cython 0.30 compat issue?  If not, I guess we could ask for input from numpy/cython folks on why this isnt equivalent

Reply:
Not at the moment. I tried using the nditer interface but that has a few quirks, notably https://github.com/numpy/numpy/issues/9808 which is unresolved upstream but I *think* a requirement for us to use it

Haven't gone far beyond that and there's probably other ways out there